src/DotNet.Status.Web/tests/DotNet.Status.Web.Tests/DependecyRegistrationTests.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.DotNet.Internal.DependencyInjection.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Dotnet.Status.Web;
+using Microsoft.Extensions.Configuration;
+using Microsoft.AspNetCore.Hosting;
+
+namespace Dotnet.Status.Web.Tests
+{
+    public class DependencyRegistrationTests
+    {
+        [Fact]
+        public void AreDependenciesRegistered()
+        {
+            Assert.True(DependencyInjectionValidation.IsDependencyResolutionCoherent(s =>
```

Comment:
This is already handled by https://github.com/dotnet/arcade-services/blob/master/src/Maestro/Maestro.Web.Tests/DependencyRegistrationTests.cs.

Actually, we should probably just merge these tests into that assembly, rather than create a second one. :-)

Reply:
Merged em!